### ISSUE TITLE
Error when attempting to clean due

### ISSUE THREAD

**Issue by AlexApps99** [Arthur-Milchior/anki-correct-due#2]
This happened on latest version of Anki for Arch Linux, only other addons I have are `Anki Simulator` and `Japanese Support`
```
Debug info:
Anki 2.1.35 (84dcaa86) Python 3.9.0 Qt 5.15.2 PyQt 5.15.2
Platform: Linux
Flags: frz=False ao=True sv=2
Add-ons, last update check: 2020-12-14 10:20:17
Add-ons possibly involved: ⁨Correcting a bug in anki which makes new card appearing in wrong order⁩

Caught exception:
Traceback (most recent call last):
  File "/home/lxb/.local/share/Anki2/addons21/127334978/redue.py", line 16, in redue
    dconfs = col.decks.dconf
AttributeError: 'DeckManager' object has no attribute 'dconf'
```
Seems like this may not be compatible with the latest version of Anki

**Comment by Arthur-Milchior** [comment]
Thanks for the bug report.

I'll need to check, anki changed so much since I created this, I don't even know whether it's still useful. I kind of hope it's not.

**Comment by AlexApps99** [comment]
I encountered the bug while getting this shared deck: [DROPPED: Genki 1 & 2 COMPLETE book 1 + JLPT N5 N4](https://ankiweb.net/shared/info/1132238466)
I eventually found a solution without using this addon, at this support page: [New cards with a due number >= 1,000,000](https://anki.tenderapp.com/kb/problems/new-cards-with-a-due-number-1000000)

It's a shame Anki doesn't have a built-in mechanism to correct for this issue, but I suppose it's just caused by a deck created in an old version

**Comment by Arthur-Milchior** [comment]
I strongly agree it is a shame. I already discussed it with Damien Elmes, and this is a point where I strongly disagree with his choice. A normal user importing a deck should never have to understand what a due value is and how to reposition themselves, that makes no sens at all.

But I'm only a contributor, so that's not my call.
The worst, in my opinion, is that it was created by a bug in anki I repaired. But while I could ensure that the bug does not occur again, that does not help with exported deck and with collection created when the bug was still present
